package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapeo/mock-data",
-  "version": "1.0.3-alpha.7",
+  "version": "2.0.0",
```

Comment:
Should this instead be done in the release script?

Reply:
I dunno know how this change slipped in there. Fixed on [700b27eca3ca348edf57b1895a3fafc696d2a3a9](https://github.com/digidem/mapeo-mock-data/pull/21/commits/700b27eca3ca348edf57b1895a3fafc696d2a3a9)